doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -329,39 +340,41 @@ Definition of storage cluster and storage pool,
 #### 6.3.13.1 Storage pool plan
 Storage pool name in each storage cluster, and nodes in Storage pool should be defined, so the storage installer will know which nodes are installing.
 
+Multiple instances are expected, each instance defines one storage node
 | Field # | type | mandatory | Instruction |
 |----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| STORAGE_CLUSTER_NAME | String | Yes |   |
-| STORAGE_POOL_NAME | String | Yes |  |
-| DEVICE_NAME | String | Yes |  |
+| STORAGE_CLUSTER_NAME | String | Yes |  Storage cluster name, which needs to follow naming rules.  |
+| STORAGE_POOL_NAME | String | Yes | Storage pool name, which needs to follow naming rules. |
+| DEVICE_NAME | String | Yes | Storage servers in each storage pool |
 
 <p align="center"><b>Table 6-3-13-1:</b> Storage Pool Plan </p>
 
 #### 6.3.13.2 Distribution storage pool info
 Storage pool information, defines the management account and network information
 
+Multiple instances are expected, each instance defines one storage pool
 | Field # | type | mandatory | Instruction |
 |----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | STORAGE_CLUSTER_NAME | String | Yes |   |
 | NODE_POOL | String | Yes |  |
 | DISK_POOL_NAME | String | Yes |  |
 | STORAGE_POOL_NAME | String | Yes |  |
-| HA_NAME_LIST | String | Yes |  |
-| AZ_NAME | List | Yes |  |
-| STORAGE_POOL_NODE_ACCOUNT | List | Yes |  |
-| MAX_QUOTA_CAPACITY | List | Yes |  |
-| STORAGE_POOL_MANAGEMENT_IP | List | Yes |   |
-| NETWORK_MASK | List | Yes |   |
-| GATEWAY | List | Yes |   |
-| VIM_USER | List | Yes |   |
-| VIM_PASSWORD | List | Yes |   |
-| PIM_USER | List | Yes |   |
-| PIM_PASSWORD | List | Yes |   |
-| STORAGE_CLUSTER_SERVICE_IP | List | Yes |   |
-| STORAGE_CLUSTER_SERVICE_GW | List | Yes |   |
-| STORAGE_CLUSTER_BACKEND_IP | List | Yes |   |
-| STORAGE_CLUSTER_BACKEND_GW | List | Yes |   |
-| BACKUP_POLICY | List | Yes |   |
+| HA_NAME_LIST | List| Yes | The corresponding HA lists for current storage pool |
+| AZ_NAME | String | Yes | The corresponding AZ for current storage pool |
+| STORAGE_POOL_NODE_CCOUNT | String | Yes | How many nodes for current storage pool |
+| MAX_QUOTA_CAPACITY | String | Yes |  |
+| STORAGE_POOL_MANAGEMENT_IP | String | Yes |  Designed virtural IP for storage pool management |
+| NETWORK_MASK | String | Yes |   |
+| GATEWAY | String | Yes |   |
+| VIM_USER | String | Yes |  VIM credential |
+| VIM_PASSWORD | String | Yes |   |
```

Comment:
Same point, we should "encourage" plain text passwords.

Reply:
Do you mean we should not?